.github/workflows/ci.yml

```diff
@@ -15,6 +15,11 @@ env:
   BUILD_CONCURRENCY: 2
   MACOS_BUILD_CONCURRENCY: 3
   TEST_TIMEOUT: 360
+  WINDOWS_TBB_DOWNLOAD_LINK: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/64957c0f-37bf-4408-909c-37ff52fe5119/w_tbb_oneapi_p_2021.11.0.49526.exe
+  WINDOWS_ICPX_DOWNLOAD_LINK: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/94e15cb5-4bcc-4fdd-91cf-0f819a54e42e/w_dpcpp-cpp-compiler_p_2024.0.2.28_offline.exe
```

Comment:
We can deal with this in another PR, but we should be able to install on windows via a package manager other than conda as well.  I think pip should offer a similar experience to apt, and allow us to get the "newest" version without updating this link each release. 

https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-windows/2024-0/install-using-package-managers.html#GUID-495DDF2D-2369-4DF5-B248-2A53123282DA

Reply:
I have a couple of caveats regarding PIP and NuGet: 
 
 - This is a speculation, but I suppose that PIP is almost identical to Conda: we will have to do the same things to configure the environment (registry patching and setting up env variables).
 - The [NuGet page](https://www.intel.com/content/www/us/en/docs/oneapi/installation-guide-windows/2024-0/nuget.html) does not contain information about DPC++ compiler.

It can be investigated later with a low priority as updating links quarterly is not much of a burden. I will create an issue.

Reply:
I don't know this for certain, but when I've previously used pip in the context of python development, it seemed to be closer to apt than conda.  I don't think it relies so much on controlling the environment itself, and hopefully wouldn't require so many tricks.  I'll try to make some time to explore this sometime in the near future.

Reply:
Thank you a lot for the assistance, Dan. I really appreciate it.

Reply:
The suggestion has been noted: https://github.com/oneapi-src/oneDPL/issues/1337